### DISCUSSION TITLE
how to delete the 'share on'

### DISCUSSION THREAD

**Discussion by UrbanSelfSim** [academicpages/academicpages.github.io#2195] [Q&A]
Hi everyone!
I want to delete the 'SHARE ON', but cannot find the code about it.
Can anyone help me with this? Many thanks.
<img width="939" alt="屏幕截图 2024-06-17 003440" src="https://github.com/academicpages/academicpages.github.io/assets/149136328/4d2ad318-5e88-4e5f-8e64-f94ad08cd869">

**Comment by rjzupkoii** [comment]
In the `_config.yml` file there are settings for each of the content types that starts at [Line 218](https://github.com/academicpages/academicpages.github.io/blob/a5ede614473523077f69ad0fd09ee527151d0e21/_config.yml#L218) of the default `_config.yml` file. If you set the line `share:` to `false` then that will remove the share buttons from the post.
